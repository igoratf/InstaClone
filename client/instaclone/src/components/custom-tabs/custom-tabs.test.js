import React from 'react';
import { shallow } from 'enzyme';
import CustomTabs from './custom-tabs.component';

describe('CustomTabs component', () => {
  it('should render CustomTabs component', () => {
    const mockProps = {
      classes: null
    }
    expect(shallow(<CustomTabs {...mockProps}/>)).toMatchSnapshot();
  })
})