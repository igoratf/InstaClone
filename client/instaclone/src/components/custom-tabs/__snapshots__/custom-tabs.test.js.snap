// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomTabs component should render CustomTabs component 1`] = `ShallowWrapper {}`;
